Makefile

```diff
@@ -82,6 +82,11 @@ unit-test: vendor
 verify:
 	@./hack/verify-all.sh
 
+## generate: Generate deepcopy,defaults,conversion
+.PHONY: generate
+generate:
+	@./hack/update-codegen.sh
+
```

Comment:
```suggestion
## update: Update all the generate
.PHONY: update
update:
	@./hack/update-all.sh

```

`./hack/update-all.sh` file needs to be created.

Also, add verify in `hack/verify-all.sh`

Reply:
I have added `update-all.sh` to run `update-codegen.sh` and rename `verify-generate` to `verify-codegen`